### PR TITLE
Move close_nonexistent_disable_issues.py queries to ClickHouse

### PR DESCRIPTION
**Pull request by clee2000**
Example run: https://github.com/pytorch/pytorch/actions/runs/11601996563/job/32305991204?pr=139296 (commented out the part that actually closes issues but the queries run)

### CODE REVIEW

File: .github/scripts/close_nonexistent_disable_issues.py [hunk 1]
```diff
@@ -3,54 +3,67 @@
...
 LOGS_QUERY = """
 with
     shas as (
         SELECT
-            push.head_commit.id as sha,
+            distinct
+            push.head_commit.id as sha
         FROM
-            commons.push
+            -- Not bothering with final here
```
Comment (huydhn) [inline .github/scripts/close_nonexistent_disable_issues.py:30]: I learn from @ZainRizvi that the use (or abuse) of having final on the query could be the reason why some of my queries run OOM.  I guess I need to go back and double check some of them

File: .github/scripts/close_nonexistent_disable_issues.py [hunk 2]
```diff
@@ -3,54 +3,67 @@
...
+    j.id in (select id from materialized_views.workflow_job_by_head_sha where head_sha in (select sha from shas))
+    and j.name like '% / test%'
     and j.name not like '%rerun_disabled_tests%'
     and j.name not like '%mem_leak_check%'
 """
 
 TEST_EXISTS_QUERY = """
 select
-    count(*) as c
+    name
```
Comment (huydhn) [inline .github/scripts/close_nonexistent_disable_issues.py:55]: Nit: Why not just count it here like the original query? (I couldn't find what is that name is used for, so I guess it's not needed?)

**Review by huydhn — APPROVED** [review]

File: .github/scripts/close_nonexistent_disable_issues.py (hunk 2 above)
Comment (clee2000) [reply .github/scripts/close_nonexistent_disable_issues.py:55]: It's faster to just find one than to find all of them

**Pull request closed without merging**